### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by gageway1**

### CODE REVIEW

File: docker-compose.yml
Comment (isthistechsupport) [inline docker-compose.yml:32]: What's the port for?

**Review by isthistechsupport — COMMENTED** [review]
LGTM but I want to change list separation before approval

File: docker-compose.yml
Comment (gageway1) [reply docker-compose.yml:32]: port is for the bot itself, i just incremented one up of the default of 80/3000.

**Comment by isthistechsupport** [comment]
Ok list separation added

File: docker-compose.yml
Comment (isthistechsupport) [reply docker-compose.yml:32]: okay I might look into it later since I'm not sure if we need to expose container ports atm

**Review by isthistechsupport — APPROVED** [review]
LGTM

**Pull request merged**
